contract_wrapper/ContractWrapper.js

```diff
@@ -78,6 +77,7 @@ class ContractWrapper {
       )
       return contractDeployed
     } catch (e) {
+      console.log(e)
```

Comment:
I think it is better to avoid `console.log()` in the code, only for debug or the commands.